.github/workflows/release-on-milestone-closed.yml

```diff
@@ -21,27 +21,17 @@ jobs:
         with:
           command-name: "laminas:automatic-releases:release"
         env:
-          "GITHUB_TOKEN": ${{ secrets.GITHUB_TOKEN }}
+          "GITHUB_TOKEN": ${{ secrets.ORGANIZATION_ADMIN_TOKEN }}
           "SIGNING_SECRET_KEY": ${{ secrets.SIGNING_SECRET_KEY }}
           "GIT_AUTHOR_NAME": ${{ secrets.GIT_AUTHOR_NAME }}
           "GIT_AUTHOR_EMAIL": ${{ secrets.GIT_AUTHOR_EMAIL }}
 
-      - name: "Create Merge-Up Pull Request"
-        uses: "laminas/automatic-releases@1.0.1"
+      - name: "Create new milestones"
+        uses: "laminas/automatic-releases@v1"
```

Comment:
This should be the same as in line 20

Reply:
Good call on the inconsistency; I've updated them both to be `@v1` though, which should negate the need to have dependabot update things I think?

Reply:
dependabot would also update to v2.

Reply:
Updates to major versions (assuming semver) may have breaking changes, let's keep it to just v1 for now I think :)